### ISSUE TITLE
Request to add relative mouse to joystick motion

### ISSUE THREAD

**Issue by akumenon** [rdoeffinger/xwa_ddraw_d3d11#3]
Hi there, just started playing X-Wing (1998) again and thanks to your work, it's very playable on Windows 10. I would however like to request relative mouse (to joystick) motion (like in FPS/arcade flight games) so I don't have to bring back the mouse to the exact location it was for level flight. This will also allow me to  map mouse movement to my Xbox controller (via steam input) so I could navigate menus using the controller as well.
Thanks!

**Comment by bazrrrr** [comment]
I play all the x-wing games with similar mouse behaviour using third party software to emulate similar movement used in the dos versions. I also blend the movement with digital movement of the arrows keys from the keyboard. The mouse and keyboard movement is listed as experimental in the cfg and as a mouse user it's not all that playable for me in it's current form. Ship movement based on mouse movement rather than position would certainly be alot more playable for me. It could be down to preference rather than an actual bug. An easy example of good mouse movement would be using this program with the vjoy driver (a program which was primarily aimed at the x-wing series):

https://github.com/raptorswing/MouseToJoystick

**Comment by rdoeffinger** [comment]
Yes, I was planning to look into it, but what I found so far made it seem rather difficult to implement in a way that actually works well.
I might try a simple implementation, but if that does not work this might simply be out of scope for this project.

**Comment by bazrrrr** [comment]
Ok thanks for exploring it further. I have used a couple of other third party programs to make it possible so if you need any help I can send those over and if you need any help with testing let me know. Preferably on discord as I'm not here often.
